Lib/fontTools/varLib/mutator.py

```diff
@@ -121,6 +122,46 @@ def instantiateVariableFont(varfont, location, inplace=False):
 		log.info("Building interpolated tables")
 		merger.instantiate()
 
+	addidef = False
+	for glyph in glyf.glyphs.values():
+		if addidef:
+			break
+		if hasattr(glyph, "program"):
+			instructions = glyph.program.getAssembly()
+			# If GETVARIATION opcode is used in bytecode of any glyph add IDEF
+			addidef = any(op.startswith("GETVARIATION") for op in instructions)
+
+	if addidef:
+		log.info("Adding IDEF to fpgm table for GETVARIATION opcode")
+		asm = []
+		if varfont.has_key('fpgm'):
```

Comment:
has_ key is a bit old-fashioned, you should use if ‘fpgm’ in varfont

Same for maxp below

Reply:
Done.